lib/board.rb

```diff
@@ -0,0 +1,35 @@
+require_relative './token'
+
+class Board
+    include Token
+    attr_accessor :size, :positions, :max_positions
+    
+    def initialize(size:)
+        @size = size
+        @max_positions = size * size
+        @positions = (1..max_positions).map {|num| num = empty }
```

Comment:
empty comes from the Token module, representing `:empty`. 

I'll be updating these "tokens" (i'll change the name!) to be constants instead, so when reading it as `num = EMPTY` be clearer?

Reply:
I would almost prefer to see something like `Token.EMPTY` so I know it refers to the Token module

Reply:
^ I'd add that this goes for any module that you are including. Prepending the module name will make it easier for others to identify where things outside of the current scope are coming from.

Reply:
I think instead of this:

```
module GameSettings
    def board_size
        {}
    end
end 

include GameSettings

GameSettings.board_size 
```

...you might be able to do this:

```
module GameSettings
  extend self
  
  def board_size
    {}
  end
end
```

Because everything in Ruby is an object, `GameSettings` can actually add those functions to its own topmost level.  Under the hood this is kind of like the code is saying `GameSettings = Module.new(GameSettings)`, and then `GameSettings.include(GameSettings)`.  That's not _exactly_ what's happening but it is pretty similar.

The end result should be that you can do stuff like this:

`@size = GameSettings.board_size`